websites/Y/Yandex.Translate/presence.ts

```diff
@@ -0,0 +1,47 @@
+const presence = new Presence({
+    clientId: "799583813582848041"
+}), browsingStamp: number = Math.floor(Date.now() / 1000);
+
+let from: string, to: string, typet: string;
+
+presence.on("UpdateData", async () => {
+
+    const presenceData: PresenceData = {
+        largeImageKey: "yt"
+      };
+    
+      if (document.location.pathname == "/") {
+        typet = "Text";
+        from = document.querySelector("#srcLangButton").innerHTML;
+        to = document.querySelector("#dstLangButton").innerHTML;
+      } else if (document.location.pathname == "/translate" || document.location.pathname == "/doc" ) {
+        typet = document.location.pathname == "/translate" ? "Website" : "Document";
+        from = document.querySelector("#srcLangButton > #sourceLangText").innerHTML;
+        to = document.querySelector("#dstLangButton > #targetLangText").innerHTML;
+      } else if (document.location.pathname == "/ocr") {
+        typet = "Image";
+        from = document.querySelector("#sourceLangButton").innerHTML;
+        to = document.querySelector("#targetLangButton").innerHTML;
+      } else {
+        typet = "Text";
+        from = "Choosing...";
+        to = "Choosing...";
+      }
+    
+      const showTime: boolean = await presence.getSetting("stamp");
+    
+      presenceData.startTimestamp = showTime ? browsingStamp : null;
+      if (presenceData.startTimestamp == null) delete presenceData.startTimestamp;
+    
+      const showType: boolean = await presence.getSetting("type");
```

Comment:
just merge these consts 🤦

Reply:
Will merge sorry, forgot to go back and look over the code after I "finished" it